src/ManageCourses.Api/Controllers/PublishController.cs

```diff
@@ -31,13 +31,16 @@ public PublishController(IDataService dataService, IEnrichmentService enrichment
         [ProducesResponseType(typeof(bool), 200)]
         [ProducesResponseType(400)]
         [ProducesResponseType(404)]
-        public async Task<ActionResult> PublishToSearchAndCompare(string instCode, string courseCode)
```

Comment:
Should this function be called PublishCourseAsync?

Reply:
Agreed. I'll change it